third_party/blink/renderer/modules/cobalt/h5vcc_runtime/h_5_vcc_runtime.cc

```diff
@@ -75,10 +67,40 @@ void H5vccRuntime::EnsureReceiverIsBound() {
       remote_h5vcc_runtime_.BindNewPipeAndPassReceiver(task_runner));
 }
 
+void H5vccRuntime::RemoteAddListener() {
```

Comment:
In general, and certainly when adding methods wholesale, they should follow the declaration order, so please move this to much further down in the file (~l.111).